session/session.go

```diff
@@ -107,15 +117,20 @@ func retrieveSession(ctx context.Context, cfg *config.Config,
 	r, err := c.User.GetSelf(p, nil)
 	if err != nil {
 		switch err.(type) {
+		case *user.GetSelfBadRequest:
+			return &Authorized{fromEnvVars: fromEnvVars, isTeam: true}, nil
```

Comment:
This seems weird to set on bad request, maybe add a comment to explain why, and maybe a TODO to explain how it should change in the future?